src/common/gui/PopupEditorDialog.cpp

```diff
@@ -139,4 +139,11 @@ void spawn_miniedit_text(char* c, int maxchars)
       strncpy(c, me.textdata, maxchars);
    }
 }
-#endif
\ No newline at end of file
+#elif __linux__
+
+void spawn_miniedit_text(char* c, int maxchars)
+{
+   // TODO
```

Comment:
Yeah should have. For this I have no immediate idea but we'll get there...

Reply:
I can certainly add a compile warning, I had just followed what was already present for the MAC_COCOA case.
As for implementing this, after a quick look, I don't think it will be trivial.  It probably needs to use Xcb to match what vstgui is using, but I am not yet sure how we get a reference to the display connection that should already be open.